pandas/tests/scalar/test_timedelta.py

```diff
@@ -502,66 +469,6 @@ def test_round(self):
         for freq in ['Y', 'M', 'foobar']:
```

Comment:
some more round that should move?

Reply:
This is the scalar test, pretty sure its in the right place.